ToopherSecondFactor/toopher-openam.js

```diff
@@ -1,183 +1,84 @@
 // vim: ts=2 sw=2 expandtab cindent
```

Comment:
agreed, and my .vimrc on my local machine makes this irrelevant.  I used a modeline here (and also in ToopherSecondFactor.java, though I hesitate to draw attention to it) because I commonly edit these files on a remote VM during the development process.